docs/UserGuide.md

```diff
@@ -382,7 +416,9 @@ On entering the `help` command, the following window will pop up:
 
 Allows you to clear all entries from the address book.
 
+<div markdown="span" class="alert alert-warning">
 :exclamation: **Caution:** This action is irreversible.
```

Comment:
![image](https://github.com/AY2324S2-CS2103T-F10-1/tp/assets/59340097/5f52a6e2-2e18-4cc7-b8d0-c45a1e5a19d6)
Caution is not bolded and users can see **, which may be due to spaces. The formatting for the boxes will probably only be seen once merged

Reply:
On my screen the display looks fine, maybe we can merge and see if the problem still persists? 
<img width="733" alt="Screenshot 2024-04-12 at 16 29 24" src="https://github.com/AY2324S2-CS2103T-F10-1/tp/assets/107994607/bf3c14c9-3d03-4466-8337-f27a2f149dab">